src/components/suggest/ch-suggest.tsx

```diff
@@ -119,6 +135,10 @@ INDEX:
 
   // 6.COMPONENT LIFECYCLE EVENTS //
 
+  componentDidLoad() {
+    this.evaluateSlotIsEmpty();
+  }
+
```

Comment:
It will be better if you check this in the componentWillLoad lifecycle method to not trigger an extra render after the initial render. To do this, you will need to use something like:
```js
componentWillLoad() {
  this.slotIsEmpty = this.el.querySelector(":scope>*");
}
```
or maybe something like this:
```js
componentWillLoad() {
  this.slotIsEmpty = !!this.el.firstElementChild;
}
```

The only problem with this is that you will have two ways to update the `slotIsEmpty` condition. So, I leave it up to you to find a way to have only one way to verify it.